tests/wallet/dao_wallet/test_dao_wallets.py

```diff
@@ -581,6 +581,7 @@ async def test_dao_proposals(self_hostname: str, three_wallet_nodes: SimulatorsA
     await asyncio.sleep(1)
     for i in range(1, num_blocks):
         await full_node_api.farm_new_transaction_block(FarmNewBlockProtocol(puzzle_hash_0))
+        await asyncio.sleep(2)
```

Comment:
`.sleep` == bad, check out `full_node_api.wait_for_wallet_synced`

Reply:
useful to know thank you